### ISSUE TITLE
Underline and overline 

### ISSUE THREAD

**Issue by rougier** [gonewest818/dimmer.el#46]
Using latest dimmer version and emacs 26.3 on OSX, I've noticed that underline/overline does not seems to be dimmed when I switch buffer. Is that only me ? If no would it be possible to dim them as well?

**Comment by gonewest818** [comment]
True, only foreground and background are dimmed right now. The overline, underline and box attributes aren’t handled, but could be. Thank you for pointing that out.

**Comment by rougier** [comment]
Ok. And by the way, many thanks for your great work.
